arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -121,8 +121,8 @@
         "tags": [
           "Registries"
         ],
-        "description": "Creates a container registry with the specified parameters.",
-        "operationId": "Registries_Create",
+        "description": "Creates or updates a container registry with the specified parameters.",
+        "operationId": "Registries_CreateOrUpdate",
```

Comment:
Per ARM's guidance, PUT a resource that already exists will update the resource (provided that immutable properties don't change). `CreateOrUpdate` matches its behavior. What is the exact concern here? It is better if we renamed it back to `Create`?